app/javascript/components/widgets/weather/daily-weather.jsx

```diff
@@ -7,14 +7,14 @@ import { colors, fontSizes, weights, spacing } from '../../../lib/theme';
 import { parseDay } from '../../../lib/datetime';
 import { WhiteText } from '../../typography';
 import { Row } from '../../row';
-import SkyIcon from './sky-icons';
+import SmallSkyIcon from './sky-icons';
```

Comment:
This should be `{ SmallSkyIcon }`. Right now is the default export, `SkyIcon` which requires `width` / `height`